tests/system/apmserver.py

```diff
@@ -1,3 +1,8 @@
+from kibana import Kibana
+from es_helper import index_smap, index_span, index_error, apm_prefix
+from es_helper import cleanup, default_pipelines
+from helper import wait_until
+from beat.beat import INTEGRATION_TESTS, TestCase, TimeoutError
```

Comment:
This is going to break, it needs to be after the `sys.path.append`. I guess this change came from autopep8?

Reply:
yes, this comes from `make fmt`. do i have the wrong version of something?

Reply:
I don't know, it happens to me too (I don't often run `make fmt` -- I just rely on vim/goimports). I've opened https://github.com/elastic/apm-server/issues/4454 to fix it.